java/build.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-mvn clean install -Dmaven.test.skip
+bazel build -c opt //java:all
```

Comment:
IIRC, this file isn't being used any more, we can just delete it.

Reply:
leave the job to developers

Reply:
if it's not used, the file should be deleted as part of this PR I'd say